backend/src/junior/entities/challenge.entity.ts

```diff
@@ -0,0 +1,11 @@
+import { Entity, Column, PrimaryColumn } from 'typeorm';
+
+@Entity()
+export class Challenge {
+
+    @PrimaryColumn({ unique: true })
+    id: string;
+
+    @Column()
+    challenge: string;
```

Comment:
would something else be more descriptive like hash? having a column be named the same as the table feels a bit weird

Reply:
In the past I have commonly known these hashes to be called "challenges"; however, I had not considered that column and table having the same name might appear confusing. I feel that as this table exists purely to store challenges this could be acceptable; however, if it felt that it's more natural to change this, I shall.